From 9a39b87b5f457ec310ffecfeb91562409d5ecefa Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:04:17 -0400
Subject: [powerpc] Rename PMU interrupts from CNT to PMI

Message-id: <1410545655-205645-29-git-send-email-dzickus@redhat.com>
Patchwork-id: 94076
O-Subject: [RHEL7 PATCH 028/626] powerpc: Rename PMU interrupts from CNT to PMI
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit e8e813ed268d90c1377f53460527c419eb52c67a
Author: Michael Ellerman <michael@ellerman.id.au>
Date:   Tue Jun 4 14:21:17 2013 +1000

    powerpc: Rename PMU interrupts from CNT to PMI

    Back in commit 89713ed "Add timer, performance monitor and machine check
    counts to /proc/interrupts" we added a count of PMU interrupts to the
    output of /proc/interrupts.

    At the time we named them "CNT" to match x86.

    However in commit 89ccf46 "Rename 'performance counter interrupt'", the
    x86 guys renamed theirs from "CNT" to "PMI".

    Arguably changing the name could break someone's script, but I think the
    chance of that is minimal, and it's preferable to have a name that 1) is
    somewhat meaningful, and 2) matches x86.

    Signed-off-by: Michael Ellerman <michael@ellerman.id.au>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/kernel/irq.c b/arch/powerpc/kernel/irq.c
index 1df7085..48c4e30 100644
--- a/arch/powerpc/kernel/irq.c
+++ b/arch/powerpc/kernel/irq.c
@@ -364,7 +364,7 @@ int arch_show_interrupts(struct seq_file *p, int prec)
   seq_printf(p, "%10u ", per_cpu(irq_stat, j).spurious_irqs);
  seq_printf(p, "  Spurious interrupts\n");
 
- seq_printf(p, "%*s: ", prec, "CNT");
+ seq_printf(p, "%*s: ", prec, "PMI");
  for_each_online_cpu(j)
   seq_printf(p, "%10u ", per_cpu(irq_stat, j).pmu_irqs);
  seq_printf(p, "  Performance monitoring interrupts\n");
-- 
1.7.1